openapi/src/json-schemas/_questionnaires_{questionnaireId}_sections_{sectionId}_answers/post/req/201.json

```diff
@@ -13,14 +13,7 @@
                     "const": "answers"
                 },
                 "attributes": {
-                    "type": "object",
-                    "additionalProperties": false,
-                    "properties": {
-                        "case-reference": {
-                            "type": "string",
-                            "pattern": "^[0-9]{2}\\\\[0-9]{6}$"
-                        }
-                    }
+                    "type": "object"
```

Comment:
Is this to make it not specific to the System section?

Reply:
Yeah exactly. The express openapi validator thing will validate the shapes of requests, however the body of the request is validated using the section schema, hence making this more generic e.g. system could ask for more than just a case reference number